lale/lib/aif360/urbis.py

```diff
@@ -84,11 +101,17 @@ def fit(self, X, y):
             lambda row: "".join([str(v) for v in row]), axis=1
         )
         assert X.shape[0] == group_and_y.shape[0]
+        # TODO: Figure out a better workaround
```

Comment:
After running ProtectedAttributeEncoder, I think should be able to unconditionally assume that the favorable labels are `set([1])`. However, I would not overwrite `self.favorable_labels`, because that would change the state of the object for future calls. Instead, why not just pass `set([1])` into the `_pick_sizes` call below?

Reply:
As discussed offline, I'm using `LabelEncoder` to encode the features separately and then construct a new set to pass to `_pick_sizes` by using the encoder's mapping and the classes in `favorable_labels`.